ui/test/dashboards/components/GraphOptionsTimeFormat.test.tsx

```diff
@@ -29,7 +28,7 @@ describe('Dashboards.Components.GraphOptionsTimeFormat', () => {
 
         expect(wrapper.find(Dropdown).exists()).toBe(true)
         expect(wrapper.find(QuestionMarkTooltip).exists()).toBe(false)
-        expect(wrapper.find(InputClickToEdit).exists()).toBe(false)
+        expect(wrapper.find('input.custom-time-format').exists()).toBe(false)
```

Comment:
not great practice to search for class names since classes can easily change. Consider adding an attribute to that input called `data-test` and find the input component by doing `wrapper.find({'data-test':'custom-time-format})`